plsync/slices.py

```diff
@@ -105,6 +105,11 @@
                                            use_initscript=True,
                                            ipv6="all",
                                            rsync_modules=['sidestream', 'npad', 'paris-traceroute']),
+    Slice(name="mlab_diff",       index=4, attrs=centos_slice_attrs+[
+                                                Attr('MeasurementLabCentos',    disk_max='100000000'),  # 100GB.
```

Comment:
100GB is probably okay, especially on all the new hardware. I just don't know off the top of my head whether there are any legacy systems without enough space. Not a blocker but just a note in case that's a possibility.